<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>goto1134</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/icons/icon.png</normaloff>:/icons/icons/icon.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="projectSplitter">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="toolTipDuration">
       <number>-3</number>
      </property>
      <property name="locale">
       <locale language="Russian" country="Russia"/>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QSplitter" name="graphicSplitter">
       <property name="locale">
        <locale language="Russian" country="Russia"/>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="locale">
         <locale language="Russian" country="Russia"/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>854</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreateDVS"/>
   <addaction name="actionLoadDVS"/>
   <addaction name="actionSaveDVS"/>
   <addaction name="separator"/>
   <addaction name="actionCheck"/>
   <addaction name="actionGarbage"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="actionStepBack"/>
   <addaction name="actionStepForward"/>
   <addaction name="separator"/>
   <addaction name="actionOptions"/>
  </widget>
  <widget class="QToolBar" name="redactorToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionCoursor"/>
   <addaction name="actionPlace"/>
   <addaction name="actionTransition"/>
   <addaction name="actionNonTerminalTransition"/>
   <addaction name="actionRegularConnection"/>
   <addaction name="actionControlConnection"/>
  </widget>
  <action name="actionCreateDVS">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/new project.ico</normaloff>:/icons/icons/new project.ico</iconset>
   </property>
   <property name="text">
    <string>Create DCN</string>
   </property>
   <property name="toolTip">
    <string>Create new DVS model</string>
   </property>
  </action>
  <action name="actionLoadDVS">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/open.ico</normaloff>:/icons/icons/open.ico</iconset>
   </property>
   <property name="text">
    <string>Load DCN</string>
   </property>
   <property name="toolTip">
    <string>Load DVS Project</string>
   </property>
  </action>
  <action name="actionSaveDVS">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/save.ico</normaloff>:/icons/icons/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save DCN</string>
   </property>
   <property name="toolTip">
    <string>Save DVS</string>
   </property>
  </action>
  <action name="actionCoursor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/coursor.ico</normaloff>:/icons/icons/coursor.ico</iconset>
   </property>
   <property name="text">
    <string>Coursor</string>
   </property>
  </action>
  <action name="actionPlace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/place.ico</normaloff>:/icons/icons/place.ico</iconset>
   </property>
   <property name="text">
    <string>Place</string>
   </property>
  </action>
  <action name="actionTransition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/terminal.ico</normaloff>:/icons/icons/terminal.ico</iconset>
   </property>
   <property name="text">
    <string>Transition</string>
   </property>
  </action>
  <action name="actionRegularConnection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/connection.ico</normaloff>:/icons/icons/connection.ico</iconset>
   </property>
   <property name="text">
    <string>Regular Connection</string>
   </property>
  </action>
  <action name="actionControlConnection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Control Connection</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/settings.ico</normaloff>:/icons/icons/settings.ico</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/play.ico</normaloff>:/icons/icons/play.ico</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/pause.ico</normaloff>:/icons/icons/pause.ico</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionGarbage">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/clean.ico</normaloff>:/icons/icons/clean.ico</iconset>
   </property>
   <property name="text">
    <string>Garbage</string>
   </property>
  </action>
  <action name="actionCheck">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/find.ico</normaloff>:/icons/icons/find.ico</iconset>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/stop.ico</normaloff>:/icons/icons/stop.ico</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionStepForward">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/step forward.ico</normaloff>:/icons/icons/step forward.ico</iconset>
   </property>
   <property name="text">
    <string>Step Forward</string>
   </property>
  </action>
  <action name="actionStepBack">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/step back.ico</normaloff>:/icons/icons/step back.ico</iconset>
   </property>
   <property name="text">
    <string>Step Back</string>
   </property>
  </action>
  <action name="actionNonTerminalTransition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/icons/nonterminal.ico</normaloff>:/icons/icons/nonterminal.ico</iconset>
   </property>
   <property name="text">
    <string>NonTerminalTransition</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
